### PR TITLE
{Seqtk}[intel-2015](bio) 

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+files_to_copy = ["seqtk", "trimadap"]
+sanity_check_paths = {
+    'files': files_to_copy,
+    'dirs': [],
+}
+modextrapaths = {
+    'PATH': ['$root'],
+}
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: less whitespace

File: easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/lh3/seqtk/archive/']
+
+buildopts = 'CC="$CC" CFLAGS="$CLFAGS"'
+
+files_to_copy = ["seqtk", "trimadap"]
+sanity_check_paths = {
```
Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: empty line in between

File: easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+name = 'seqtk'
+version = 'sgdp'
+
+homepage = 'https://github.com/lh3/seqtk/'
+description = """Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip."""
```
Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: fix long line

File: easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+name = 'seqtk'
```
Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: empty line above

**Comment by JensTimmerman** [comment]
please rereview

File: easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+files_to_copy = ["seqtk", "trimadap"]
+
+sanity_check_paths = {
+    'files': files_to_copy,
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': ['$root'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: please use `%(installdir)s` here; `$root` is specific to Tcl modules, and we're close to adding support for Lua module files

Comment (boegel) [inline easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb]: ah, wait, just use `''` instead (`eb` doesn't like absolute paths being used here)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.9
See https://gist.github.com/b6e98dc9be523cbb53a5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JensTimmerman!

**Pull request merged**